## Import libraries

In [26]:
from scipy.stats import t
import math

### Hypothesis test

In [27]:
(130.1 - 120)/(21.21/math.sqrt(100))

4.761904761904759

### T-Student distribution

In [28]:
tc = +- t.ppf(0.95, df=99)
tc

-1.6603911559963895

yes the group is significantly different from the regular 
population. The range for the null hypothesis range from -1.660 - 1.660 but the result is 4.76190. So it is out of range.